spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/pubsub/it/PubSubAutoConfigurationIntegrationTests.java

```diff
@@ -37,11 +39,26 @@
 public class PubSubAutoConfigurationIntegrationTests {
 
 	private static final Log LOGGER = LogFactory.getLog(PubSubTemplateIntegrationTests.class);
+
 	private static GcpProjectIdProvider projectIdProvider;
+
 	private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
-			.withPropertyValues("spring.cloud.gcp.pubsub.subscriber.max-ack-extension-period=0",
+			.withPropertyValues(
 					"spring.cloud.gcp.pubsub.subscription.test-sub-1.executor-threads=3",
```

Comment:
Minor nit: For this test case, can you group all the `test-sub-1` properties together and the `test-sub-2` properties together?

Reply:
This definitely improves readability, thanks!